yarn.lock

```diff
@@ -11611,6 +11611,13 @@ ovh-angular-user-pref@^0.3.1:
   resolved "https://registry.yarnpkg.com/ovh-angular-user-pref/-/ovh-angular-user-pref-0.3.1.tgz#07f5792513651f3262ae4879c174423649b08307"
   integrity sha1-B/V5JRNlHzJirkh5wXRCNkmwgwc=
 
+ovh-api-services@^9.26.0:
```

Comment:
We're having two versions of `ovh-api-services`, imo we should keep the latest one